content/docs/command-reference/exp/run.md

```diff
@@ -180,27 +195,36 @@ experiment we just ran (`exp-44136`).
 ## Example: Modify parameters on-the-fly
 
 `dvc exp run--set-param` (`-S`) saves you the need to manually edit the params
-file before running an experiment.
+file before running an experiment. It can override (`foo.bar=value`), append
+(`+foo.bar=value`), or remove (`~foo.bar`) parameters.
+
+<admon type="tip">
+
+DVC supports the
+[Hydra Override Grammar](https://hydra.cc/docs/advanced/override_grammar/basic/),
+with the exception of patterns that modify Hydra's defaults list.
+
+</admon>
+
+You can optionally provide a prefix `[<filename>:]` in order to edit a specific
+`dvc params` file. If not provided, `params.yaml` will be used as default.
 
 ```dvc
-$ dvc exp run -S prepare.split=0.25 -S featurize.max_features=2000
+$ dvc exp run -S custom_file.json:+newparam=foo
```

Comment:
Same as above: 
I find `newparam=foo` too generic and it's not clear why this syntax is useful